docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[release-notes-header-8.3.0]]
+== 8.3
+
+[discrete]
+[[release-notes-8.3.0]]
+== 8.3.0
+
+[discrete]
+[[breaking-changes-8.3.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+* Adds confirmation modal with docs link when updating rules if V1/V2 jobs are installed ({pull}128334[#128334]).
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.3.0]]
+==== Features
+* Renames Endpoint Security integration to "{endpoint-cloud-sec}" ({pull}132752[#132752]).
+* Adds a “Cases” widget that shows cases by status to the Detection & Response page ({pull}130670[#130670]).
+* The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts ({pull}128335[#128335], {pull}129021[#129021], {pull}128087[#128087], {pull}131828[#131828], {pull}131029[#131029]).
+* Adds a *User risk* tab to the User details flyout ({pull}130256[#130256]).
+* Adds an *Authentications* tab to the User details flyout ({pull}129456[#129456]).
+* Adds the ability to investigate Osquery results in Timeline ({pull}128596[#128596]).
+* Allows multiple alerts to be added to a case ({pull}130958[#130958]).
+* Adds the option to delete case comments from a case ({pull}130254[#130254]).
+* Provides an option to select a severity level for a case ({pull}131626[#131626]).
+* Adds the *Alerts* tab to cases ({pull}131883[#131883]).
```

Comment:
I think is better to say that you can view all alerts attached to a case so the users can understand better what this feature is about. Also, the feature is experimental. Maybe it is worth mentioning that also. What do you think?

Reply:
Ah, I see what you're saying about being more descriptive @cnasikas. How about:
Adds the *Alerts* tab to cases, which is an experimental feature that allows users to inspect attached alerts.

Reply:
Perfect!